Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: astronomer
-version: 0.18.0
-appVersion: 0.18.0
+version: 0.18.1
+appVersion: 0.18.1
```

Comment:
```suggestion
version: 0.18.0
appVersion: 0.18.0
```

Reply:
How do we pick up the Houston API changes if it's not going off of the version number here?